Tests/Engine/ModuleHelp.Tests.ps1

```diff
@@ -214,15 +214,19 @@ Get-Module $ModuleName | Remove-Module
 Import-Module $ModuleName -RequiredVersion $RequiredVersion -ErrorAction Stop
 $ms = $null
 $commands =$null
-$paramBlackList = @()
+$paramBlackList = @(
+	'AttachAndDebug' # Reason: When building with DEGUG configuration, an additional parameter 'AttachAndDebug' will be added to Invoke-ScriptAnalyzer and Invoke-Formatter, but there is no Help for those, as they are not intended for production usage.
+)
 if ($PSVersionTable.PSVersion -lt [Version]'5.0.0') {
 	$ms = New-Object -TypeName 'Microsoft.PowerShell.Commands.ModuleSpecification' -ArgumentList $ModuleName
 	$commands = Get-Command -Module $ms.Name
 	$paramBlackList += 'SaveDscDependency'
 }
 else {
 	$ms = [Microsoft.PowerShell.Commands.ModuleSpecification]@{ ModuleName = $ModuleName; RequiredVersion = $RequiredVersion }
-	$commands = Get-Command -FullyQualifiedModule $ms -CommandType Cmdlet,Function,Workflow # Not alias
+	# Because on Windows Powershell we have workflow, we need to include it there, but in pwsh, we can't. This makes sure this works on both.
+	$commandTypes = ([System.Enum]::GetNames("System.Management.Automation.CommandTypes") -match "^Cmdlet$|^Function$|^Workflow$")
+	$commands = Get-Command -FullyQualifiedModule $ms -CommandType $commandTypes
```

Comment:
`$ms` is just `PSScriptAnalyzer` in this case if I read correctly and PSSA only exports 3 cmdlets, no function/alias/workflow at all, therefore why not simplify it to just
```pwsh
$commands = Get-Command -FullyQualifiedModule $ms
```

Reply:
yes, makes a lot of sense. will do that.